docs/CHANGELOG.asciidoc

```diff
@@ -10,10 +10,496 @@
+
+*Impact* +
+Configure the settings in https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[ECS].
+====
+
+[float]
+[[machine-learning-breaking-changes-8.0.0-beta1]]
+==== Machine learning
+      
+[discrete]
+[[breaking-115444]]
+.Grants access to machine learning features when base privileges are used
+[%collapsible]
+====
+*Details* +
+Machine learning features are included as base privileges. For more information, refer to {kibana-pull}115444[#115444].
+
+*Impact* +
+If you do not want to grant users privileges to machine learning features, update <<xpack-security,*Users* and *Roles*>>.
+====
+
+[float]
+[[maps-breaking-changes-8.0.0-beta1]]
+==== Maps
+
+[discrete]
+[[breaking-116184]]
+.Removes proxyElasticMapsServiceInMaps
+[%collapsible]
+====
+*Details* +
+The `map.proxyElasticMapsServiceInMaps` setting has been removed. For more information, refer to {kibana-pull}116184[#116184].
+
+*Impact* +
+Install the on-prem version of the <<maps-connect-to-ems,Elastic Maps Service>>, which is a Docker service that resides in the Elastic Docker registry, in an accessible location on your internal network. When you complete the installation, update kibana.yml to point to the service.
+====
+
+[float]
+[[operations-breaking-changes-8.0.0-beta1]]
+==== Operations 
+
+[discrete]
+[[breaking-111535]]
+.Removes environment variables
+[%collapsible]
+====
+*Details* +
+The `CONFIG_PATH` and `DATA_PATH` environment variables have been removed. For more information, refer to {kibana-pull}111535[#111535].
+
+*Impact* +
+Replace the `CONFIG_PATH` environment variable with `KBN_PATH_CONF`, and replace `DATA_PATH` with the `path.data` setting.
+====
+
+[float]
+[[platform-breaking-changes-8.0.0-beta1]]
+==== Platform 
+      
+[discrete]
+[[breaking-114379]]
+.Removes supports for csp.rules configuration
+[%collapsible]
+====
+*Details* +
+Support for the `csp.rules` configuration property has been removed. For more information, refer to {kibana-pull}114379[#114379].
+
+*Impact* +
+Configuring the default `csp.script_src`, `csp.workers_src`, and `csp.style_src` values is not required.
+====
+
+[discrete]
+[[breaking-113653]]
+.Removes unused and renamed deprecated core settings and deprecated settings from core plugins
+[%collapsible]
+====
+*Details* +
+The deprecation notice for `server.cors` has changed from `level:critical` to `level:warning`. 
+
+Support for the following configuration settings has been removed:
+
+* `newsfeed.defaultLanguage`
+* `xpack.banners.placement`
+* `cpu.cgroup.path.override`
+* `cpuacct.cgroup.path.override`
+* `server.xsrf.whitelist`
+* `xpack.xpack_main.xpack_api_polling_frequency_millis`
+* `KIBANA_PATH_CONF`
+
+For more information, refer to {kibana-pull}113653[#113653].
+
+*Impact* +
+* `newsfeed.defaultLanguage` -- Newsfeed items are retrieved based on the browser locale and default to English
+* `xpack.banners.placement` has changed value from 'header' to 'top'
```

Comment:
nit: I might rephrase this to something like "The 'header' value provided to the xpack.banners.placement config has been renamed to 'top'"